sw/z80/utils/import2core.z80

```diff
@@ -1,5 +1,11 @@
 ; Core - Shared by Versions for all OS's
 
+; Bug found by: hallorant
+; A paramater parsing bug was found in import2core.z80 that has been fixed.  The
```

Comment:
typo: parameter

Reply:
Well spotted thank you! ;-) I just copied his comments over!